src/base/bittorrent/torrentimpl.cpp

```diff
@@ -967,7 +967,13 @@ int TorrentImpl::queuePosition() const
 
 QString TorrentImpl::error() const
 {
-    return QString::fromStdString(m_nativeStatus.errc.message());
+    if (m_nativeStatus.errc)
+        return QString::fromStdString(m_nativeStatus.errc.message());
+
+    if (m_nativeStatus.flags & lt::torrent_flags::upload_mode)
+        return tr("There's not enough space on disk.");
```

Comment:
Maybe add - `There's not enough space on disk. Torrent is currently in upload only mode.`
Long message won't cause issues since the full message will only be displayed on hover action.

![img](https://user-images.githubusercontent.com/79678786/115495368-4c583300-a289-11eb-9cfc-cf1dcccd5ba8.PNG)

Reply:
Done.